testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesTest.java

```diff
@@ -1373,10 +1372,119 @@ public void testSecureRequestObjectExecutor() throws Exception {
         requestObject = createValidRequestObjectForSecureRequestObjectExecutor(clientId);
         registerRequestObject(requestObject, clientId, Algorithm.ES256, false);
         oauth.openLoginForm();
-        assertEquals(SecureRequestObjectExecutor.INVALID_REQUEST_OBJECT, oauth.getCurrentQuery().get(OAuth2Constants.ERROR));
+        assertEquals(OAuthErrorException.INVALID_REQUEST_OBJECT, oauth.getCurrentQuery().get(OAuth2Constants.ERROR));
         assertEquals("Request object not encrypted", oauth.getCurrentQuery().get(OAuth2Constants.ERROR_DESCRIPTION));
     }
 
+    @Test
+    @EnableFeature(value = Profile.Feature.PAR, skipRestart = true)
```

Comment:
@pedroigor I think this is not required anymore as PAR is enabled by default. I am not sure about side-effects of this with PAR enabled by default (I guess PAR might be disabled after this test, which can have more side-effects). Is it ok if you rebase to latest master and remove this line from this PR?